test/e2e/utils/pod.go

```diff
@@ -18,18 +18,103 @@ package utils
 
 import (
 	"context"
+	"flag"
+	"sync"
 	"time"
 
+	"github.com/onsi/ginkgo"
+
 	appsv1 "k8s.io/api/apps/v1"
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/apimachinery/pkg/util/wait"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/kubectl/pkg/util/podutils"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/utils/pointer"
+)
+
+var pullIfNotPresent = flag.Bool("nfd.pull-if-not-present", false, "Pull Images if not present - not always")
+
+const (
+	CentosImage = "quay.io/centos/centos:8"
```

Comment:
Hmm, this didn't catch my eye until now, but I realized that the image we use probably does not matter at all (just sleeping and waiting in our test cases). Would it be simpler to just use `k8s.gcr.io/pause`. Would simplify podSpec (no need to specify `Command:` and also virtually eliminate the need to re-pull (plus the image size when pulling is a lot smaller). WDYT?

Reply:
sure thing! changed as suggested.